build-tools/packages/build-tools/src/common/fluidRepo.ts

```diff
@@ -130,6 +135,28 @@ export type PreviousVersionStyle =
 	| "~previousMajor"
 	| "~previousMinor";
 
+/**
+ * Configuration for the `generate:releaseNotes` command.
+ */
+export interface ReleaseNotesConfig {
+	sections: ReleaseNotesSection[];
+}
+
+/**
+ * Configuration for a release notes section.
+ */
+export interface ReleaseNotesSection {
+	/**
+	 * A short, one-word name for the section. This is used as a key.
+	 */
+	name: string;
```

Comment:
Given the docs, would it make more sense to just call this "key"? Also, does it actually have to be 1 word? Or just something unique.

Reply:
Technically it needs to be a valid YAML identifier. So no, not strictly one word, but the intent is that they're short. I don't like using 'key' in config files if I can avoid it. Can probably just remove the note about it being used as a key.

Reply:
If we remove the "use as a key" terminology, we should at least keep "they should be unique", I think? To enforce that I might even directly do something like:
```typescript
export interface ReleaseNotesSection {
  heading: string
};
export interface ReleaseNotesConfig {
  sections: Record<string, ReleaseNotesSection>;
}
```

Reply:
I tried the Record approach but I don't like the extra type complexity. It feels too complex for the use-case. If needed
I can check for duplicates at runtime, but I think docs will be enough. I anticipate 7-10 sections max at any given
time, probably fewer.

Reply:
Well, I went with a modified Record approach after all, since I anticipate a section may have more fields eventually.